modules/userId/index.js

```diff
@@ -126,9 +126,9 @@
  */
 
 /**
-  * @typedef {Object} RefreshUserIdsOptions
```

Comment:
this whole file just seems like whitespace changes now...can you just revert them all since there's no functional change?